themes/vocabulary_theme/templates/layout.html

```diff
@@ -64,7 +61,8 @@
             Blog
           </a>
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link is-arrowless {% if this.is_child_of('/contributing-code') %} is-active{% endif%}">Contribute<i class="icon caret-down"></i></a>
+            <a class="navbar-link is-arrowless {% if this.is_child_of('/contributing-code') %} is-active{% endif%}">Contribute<i
+                class="icon caret-down"></i></a>
```

Comment:
Please use linebreaks at right locations.
```suggestion
            <a class="navbar-link is-arrowless {% if this.is_child_of('/contributing-code') %} is-active{% endif%}">
              Contribute <i class="icon caret-down"></i>
            </a>
```